windows/src/support/texteditor/Makefile

```diff
@@ -0,0 +1,26 @@
+#
+# Keyman Text Editor Makefile
+#
+
+!include ..\..\Defines.mak
+
+build:
+    $(MSBUILD) editor.vcxproj $(MSBUILD_BUILD)
+    $(COPY) Release\editor.exe $(PROGRAM)\support
+    $(COPY) Release\editor.pdb $(DEBUGPATH)\support
+
+clean: def-clean
+    $(MSBUILD) $(MSBUILD_CLEAN) editor.sln
+    rmdir /s /q release
+
+signcode:
+    $(SIGNCODE) /d "Keyman Engine x64" $(PROGRAM)\support\editor.exe
+
+install:
+    $(COPY) $(PROGRAM)\support\editor.exe "$(INSTALLPATH_KEYMANENGINE)"
```

Comment:
I think we are going to need 32-bit and 64-bit versions of this to verify integration. See my earlier comment about file system layout also.

Reply:
I have made step one building 64 and 32-bit versions. However I need to workout how we can integrate in to the make system to build both.

Reply:
Solved I will just build both by supply the platform parameter in the make file at the TextEditor level for now.
```    
$(MSBUILD) editor.vcxproj $(MSBUILD_BUILD) /p:Platform=x86
$(MSBUILD) editor.vcxproj $(MSBUILD_BUILD) /p:Platform=x64
```